src/lib/audio/audio-util.js

```diff
@@ -0,0 +1,27 @@
+const computeRMS = function (samples, scaling = 0.55) {
```

Comment:
It is now that we have a test system! Check 8c2ced0. 

Thanks for pushing on that, writing those tests helped find an off-by-one error in the chunking code.